newIDE/app/src/GameDashboard/GameAnalyticsPanel.js

```diff
@@ -169,6 +184,20 @@ export const GameAnalyticsPanel = ({
                       {Math.round(chartData.overview.bounceRatePercent)}% bounce
                       rate
                     </Trans>
+                    <Tooltip
+                      title={
+                        <Text>
+                          <Trans>
+                            Percentage of people who leave before 60 seconds
+                            including loading screens. The smaller the number,
+                            the better — it means more people are interested and
+                            stay longer.
+                          </Trans>
```

Comment:
Stating the obvious only adds noise. People won't read it if it's 2 long sentences.
```suggestion
                          <Trans>
                            Percentage of people who leave before 60 seconds
                            including loading screens.
                          </Trans>
```

Reply:
It's not a technical thing. I bet they understand that people leaving their game before 1 minute is a bad thing.